fastlane/lanes/localization.rb

```diff
@@ -122,49 +127,78 @@
   # @called_by complete_code_freeze
   #
   lane :generate_strings_file_for_glotpress do |options|
-    cocoapods
-
-    # On top of fetching the latest Pods, we also need to fetch the source for the Gutenberg code.
-    # To get it, we need to manually clone the repo, since Gutenberg is distributed via XCFramework.
-    # XCFrameworks are binary targets and cannot extract strings via genstrings from there.
-    config = gutenberg_config!
-
-    ref_node = config[:ref]
-    UI.user_error!('Could not find Gutenberg ref to clone the repository in order to access its strings.') if ref_node.nil?
-
-    ref = ref_node[:tag] || ref_node[:commit]
-    UI.user_error!('The ref to clone Gutenberg in order to access its strings has neither tag nor commit values.') if ref.nil?
-
-    github_org = config[:github_org]
-    UI.user_error!('Could not find GitHub organization name to clone Gutenberg in order to access its strings.') if github_org.nil?
-
-    repo_name = config[:repo_name]
-    UI.user_error!('Could not find GitHub repository name to clone Gutenberg in order to access its strings.') if repo_name.nil?
-
-    # Create a temporary directory to clone Gutenberg into.
-    # We'll run the rest of the automation from within the block, but notice that only the Gutenbreg cloning happens within the temporary directory.
-    gutenberg_clone_name = 'Gutenberg-Strings-Clone'
+    # We jump in the tempdir immediately, even though we might not need it.
+    # It's just simpler to rely on the Ruby block API for it than to manage it by hand.
```

Comment:
❓  Is this comment really accurate?

From reading the rest of the code, it seems that we actually _don't_ jump in the tempdir immediately, but instead only do the `Dir.chdir` later in the code (line 173) when we're ready to clone the repo. So this comment seems misleading?

Besides, I'd suggest to also move the `Dir.mktmpdir` on line 173 just before the repo cloning, otherwise we will create a useless tempdir needlessly if a custom `gutenberg_path` option is provided and we don't need any cloning to begin with.

Reply:
>❓ Is this comment really accurate?
>
> From reading the rest of the code, it seems that we actually don't jump in the tempdir immediately,

That's correct. I even went back and did a `git commit --fixup` to replace "jump" with "create" but I clearly messed up a conflict resolution when interactive rebasing it...